packages/edge-gateway/wrangler.toml

```diff
@@ -64,6 +67,7 @@ kv_namespaces = [{ binding = "DENYLIST", id = "f4eb0eca32e14e28b643604a82e00cb3"
 [env.staging.vars]
 IPFS_GATEWAYS = "[\"https://ipfs.io\", \"https://cf.nftstorage.link\", \"https://pinata.nftstorage.link\"]"
 GATEWAY_HOSTNAME = 'ipfs-staging.nftstorage.link'
+EDGE_GATEWAY_API_URL = 'https://api.nftstorage.link'
```

Comment:
Should be staging?

Reply:
Can we get rid of this? It would be nice not to need this URL to match up with the binding config as it causes an invariant that we humans must ensure we configure correctly. The binding config should be enough.

Reply:
Sadly we cannot :( Fetch fails if we do not provide a full URL. We can provide something random like example.com, but feeling like keeping the real URL is better (and also allows future move out of bindings by just changing code from env.API.fetch to fetch)